src/boot/bootctl.c

```diff
@@ -1977,10 +1975,10 @@ static int verb_list(int argc, char *argv[], void *userdata) {
 
 static int install_random_seed(const char *esp) {
         _cleanup_(unlink_and_freep) char *tmp = NULL;
-        _cleanup_free_ void *buffer = NULL;
+        unsigned char buffer[RANDOM_EFI_SEED_SIZE];
```

Comment:
we usually use "uint8_t" for cases like this.

Reply:
Oh good I like that better too.

Reply:
not addressed yet?